tests/js/spec/components/dropdownLink.spec.jsx

```diff
@@ -76,8 +76,10 @@ describe('DropdownLink', function() {
         const evt = document.createEvent('HTMLEvents');
         evt.initEvent('click', false, true);
         document.body.dispatchEvent(evt);
-        wrapper.update();
-        expect(wrapper.find('li')).toHaveLength(0);
+        setTimeout(() => {
```

Comment:
In tests use `await tick()` instead of timeouts. Otherwise the test will finish before the expect can be called.